### PR TITLE
empty state illustration image for pinned messages

### LINKED ISSUE #17522: Posts & attachments - Pinned messages drawer - Add empty state illustration
## Problem

Missing implementation for the empty state for pinned messages. There should be an illustration.

<table>
  <thead>
    <tr>
      <th>Actual</th>
      <th>Expected</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>

### Resources

Reported at: https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6130-58226&mode=design&t=ZiIFjqxnna5FtOFo-0

Figma specs: https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=3513-380099&mode=design&t=tqru9be4o4mYQhQm-0

### PR DESCRIPTION
**Pull request by jo-mut**
fixes https://github.com/status-im/status-mobile/issues/17522

### Summary
Pr add empty state to pinned messages in channel and chat

**Light Theme** &  **Dark Theme**

### CODE REVIEW

File: src/status_im/common/resources.cljs [hunk 1]
```diff
@@ -84,7 +84,10 @@
     :dark  (js/require "../resources/images/ui2/no-permissions-dark.png")}
    :sweating-man
    {:light (js/require "../resources/images/ui2/sweating-man-light.png")
-    :dark  (js/require "../resources/images/ui2/sweating-man-dark.png")}})
+    :dark  (js/require "../resources/images/ui2/sweating-man-dark.png")}
+   :no-pinned-messages
+   {:light (js/require "../resources/images/ui2/no-pinned-messages-light.png")
```
Comment (J-Son89) [inline src/status_im/common/resources.cljs:90]: 👍

File: src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs [hunk 2]
```diff
@@ -2,9 +2,11 @@
   (:require
     [quo.core :as quo]
     [quo.foundations.colors :as colors]
+    [quo.theme :as quo.theme]
```
Comment (J-Son89) [inline src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs]: [quo.theme]

File: src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs [hunk 3]
```diff
@@ -83,4 +77,11 @@
          :key-fn      list-key-fn
          :separator   [quo/separator {:style {:margin-vertical 8}}]}]
        [empty-pinned-messages-state
-        {:community? (boolean community)}])]))
+        {:community? (boolean community)
+         :theme      theme}])]))
+
+(defn- internal-pinned-messages
+  [params]
+  [:f> f-pinned-messages params])
```
Comment (J-Son89) [inline src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs:85]: you shouldn't need to make this a functional component to work with `with-theme` wrapper btw - the with-theme method should handle this already 🤔

**Review by J-Son89 — APPROVED** [review]

**Review by mohsen-ghafouri — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

**Comment by mariia-skrypnyk** [comment]
Hi @jo-mut !
Thanks for your PR!

I am wondering if this empty state should contain "Just keep pinning" text and the part with community name and channel that I've marked here:
Screenshot 2024-01-31 at 19 25 31

Now you have implemented an updated image that looks fine:
Screenshot 2024-01-31 at 19 21 54

**Comment by jo-mut** [comment]
@mariia-skrypnyk good question. Now that you asked this I think design @pedro-et  can clarify. 

For community channels i based on this figma https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=3513%3A380099&mode=design&t=PrimwZf6ymyE15om-1 and for messages i just left it was it was before as i did not find the scenario that is specific to the pinned messages of 1-1 chats

I wonder if there is something am missing?

**Review by Parveshdhull — APPROVED** [review]

**Comment by mariia-skrypnyk** [comment]
> @mariia-skrypnyk good question. Now that you asked this I think design @pedro-et can clarify.
> 
> For community channels i based on this figma https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=3513%3A380099&mode=design&t=PrimwZf6ymyE15om-1 and for messages i just left it was it was before as i did not find the scenario that is specific to the pinned messages of 1-1 chats
> 
> I wonder if there is something am missing?

Asked the design team.

**Comment by mariia-skrypnyk** [comment]
@jo-mut I am going to move it for @Francesca-G review for now.

**Review by Francesca-G — APPROVED** [review]
Looks good to me ✨

**Pull request merged**